packages/RichTextEditing-Core.package/RichTextEditorModel.class/instance/initializeCompletionController.st

```diff
@@ -0,0 +1,6 @@
+initialize-release
+initializeCompletionController
+
+	^ self wantsAutocompletion
+		ifTrue: [self addDependent: self createCompletionController]
+		ifFalse: [nil]
```

Comment:
This is always called when a new SquidWord is opened?

Reply:
Yes should be